rtf

```diff
@@ -298,7 +301,18 @@ class Vagrant(Host):
   def start(self):
     """ Setup and run this host """
     proxy = have_proxy()
-    os.system(proxy + "vagrant up " + self.hostid + " --provider="+self.provider)
+    if (proxy==None):
```

Comment:
I'd say this is over engineered, the empty proxy variable is sufficient for all the cases